src/imagePickerIOS.rei

```diff
@@ -0,0 +1,15 @@
+let canUseCamera: (unit => unit) => unit;
```

Comment:
`can*` functions' callbacks here, have booleans as the first arg (see the [implementation here](https://github.com/facebook/react-native/blob/master/Libraries/CameraRoll/RCTImagePickerManager.m#L42))